framework/test/jest/unit/specs/controller/helpers/validator/keywords/env.spec.js

```diff
@@ -36,6 +36,44 @@ describe('validator keyword "env"', () => {
 		expect(data.prop1).toBe('changedValue');
 	});
 
+	it('should accept env variable if specified as integer and format accordingly', () => {
+		const envSchemaWithOutFormatter = {
+			type: 'object',
+			properties: {
+				prop2: {
+					type: 'integer',
+					env: 'PROP2',
+				},
+			},
+		};
+
+		const data = { prop2: '999' };
+		process.env.PROP2 = '999';
```

Comment:
process.env need to be cleaned up in afterEach

Reply:
Fixed!